ansible/configs/ansible-multitier-rhel8/post_software.yml

```diff
@@ -84,6 +84,15 @@
           vars:
             ACTION: create
 
+    - name: Set workstation public ip address user data
+      agnosticd_user_info:
+        data:
+          rhel8lab_workstation_public_ip_address: "{{ groups['bastions'][0] | regex_replace('\\..*$') }}.{{ guid }}{{ subdomain_base_suffix }}"
+          #bookbag_os_auth_url: "{{ osp_auth_url }}"
+          #bookbag_os_username: "{{ guid }}-user"
+          #bookbag_os_password: "{{ hostvars.localhost.heat_user_password }}"
+          #bookbag_os_project_name: "{{ guid }}-project"
```

Comment:
Please do not include commented code.

Reply:
I think if there is a short term plan to enhance the playbook but immediate needs preclude all the necessary testing etc than commenting out code causes no harm? It avoids unnecessary re-work. It's a very common pattern to implement say 3 out of 4 tasks/functions/features and comment out the element of WIP as you push the completed into production. If on the other hand it will never be used I agree.